3-extra/1-count-words.js

```diff
@@ -25,10 +25,21 @@
 
 function countWords(string) {
   const wordCount = {};
-
+  if (!string || string.trim().length === 0) {
+    return wordCount;
+  }
+  let strArr = string.split(" ");
+  strArr.map((word) => {
+    if (wordCount[word] === undefined) {
+      wordCount[word] = 1;
+    } else {
+      wordCount[word] = wordCount[word] + 1;
+    }
+  });
   // write code here
 
   return wordCount;
+  
```

Comment:
great job making it onto the extra tasks and on having passing tests! A shorthand way of doing `wordCount[word] = wordCount[word] + 1;` would be `wordCount[word] += 1`;